lib/import_csv_to_big_query.rb

```diff
@@ -0,0 +1,25 @@
+require 'google/cloud/bigquery'
+require 'rollbar'
+require 'export_tables_to_cloud_storage'
+
+class ImportCSVToBigQuery
+  def load(bigquery: Google::Cloud::Bigquery.new)
+    dataset = bigquery.dataset ENV['BIG_QUERY_DATASET']
+
+    ExportTablesToCloudStorage::TABLES.each do |table_csv|
+      import_csv_uri = "gs://#{ENV['CLOUD_STORAGE_BUCKET']}/csv_export/#{table_csv.underscore}.csv"
+      table_id = import_csv_uri.split('/').last.sub('.csv', '')
```

Comment:
Isn't it easier to just use `table_csv.underscore` here instead of doing this? You could put that in a local variable to not to repeat it

Reply:
I've updated this line to use `table_csv.underscore` but leaving it assigned to `table_id` for clarity that it represents that parameter in big query